<ui version="4.0" >
 <author>Tomeu Borras Riera</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>selectorbase</class>
 <widget class="QMainWindow" name="selectorbase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>211</width>
    <height>96</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>BulmaGes</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_bulmafact" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>10000</width>
          <height>10000</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>BulmaFact</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../bulmages.qrc" >:/Genericos48x48/images/pendientes/bulmafact.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>150</width>
          <height>16</height>
         </size>
        </property>
        <property name="shortcut" >
         <string>Alt+F</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
        <property name="autoRepeat" >
         <bool>false</bool>
        </property>
        <property name="default" >
         <bool>false</bool>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bulmatpv" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>BulmaTPV</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../bulmages.qrc" >:/Genericos48x48/images/pendientes/bulmatpv.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>150</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bulmacont" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>BulmaCont</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../bulmages.qrc" >:/Genericos48x48/images/pendientes/bulmacont.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>150</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_galopin" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Galopin</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_iglues" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Iglues</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_bcontaweb" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>BContaWeb</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line2" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Empresa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombreempresa" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>238</red>
            <green>234</green>
            <blue>222</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>252</blue>
           </color>
           <color>
            <red>85</red>
            <green>85</green>
            <blue>82</blue>
           </color>
           <color>
            <red>198</red>
            <green>198</green>
            <blue>191</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
           <color>
            <red>238</red>
            <green>238</green>
            <blue>230</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>221</green>
            <blue>118</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>192</blue>
           </color>
           <color>
            <red>128</red>
            <green>0</green>
            <blue>128</blue>
           </color>
           <color>
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </active>
          <inactive>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>238</red>
            <green>234</green>
            <blue>222</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>252</blue>
           </color>
           <color>
            <red>85</red>
            <green>85</green>
            <blue>82</blue>
           </color>
           <color>
            <red>198</red>
            <green>198</green>
            <blue>191</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
           <color>
            <red>238</red>
            <green>238</green>
            <blue>230</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>255</red>
            <green>221</green>
            <blue>118</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>192</blue>
           </color>
           <color>
            <red>128</red>
            <green>0</green>
            <blue>128</blue>
           </color>
           <color>
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </inactive>
          <disabled>
           <color>
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
           <color>
            <red>238</red>
            <green>234</green>
            <blue>222</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>252</blue>
           </color>
           <color>
            <red>85</red>
            <green>85</green>
            <blue>82</blue>
           </color>
           <color>
            <red>198</red>
            <green>198</green>
            <blue>191</blue>
           </color>
           <color>
            <red>198</red>
            <green>198</green>
            <blue>191</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>0</blue>
           </color>
           <color>
            <red>238</red>
            <green>238</green>
            <blue>230</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
           <color>
            <red>212</red>
            <green>184</green>
            <blue>98</blue>
           </color>
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
           <color>
            <red>0</red>
            <green>0</green>
            <blue>192</blue>
           </color>
           <color>
            <red>128</red>
            <green>0</green>
            <blue>128</blue>
           </color>
           <color>
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </disabled>
         </palette>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="seleccionaempresa" >
        <property name="toolTip" >
         <string>Seleccionar Empresa</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../bulmages.qrc" >:/BulmaCont32x32/images/pendientes/i_empresa.xpm</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configura" >
        <property name="toolTip" >
         <string>Configuración</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../../../bulmages.qrc" >:/BulmaCont32x32/images/pendientes/i_configuracion.xpm</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>51</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="salir" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Salir</string>
        </property>
        <property name="text" >
         <string>S&amp;incronizar</string>
        </property>
        <property name="default" >
         <bool>false</bool>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>salir</tabstop>
  <tabstop>m_iglues</tabstop>
  <tabstop>m_bulmatpv</tabstop>
  <tabstop>m_bulmacont</tabstop>
  <tabstop>m_galopin</tabstop>
  <tabstop>m_bcontaweb</tabstop>
  <tabstop>m_bulmafact</tabstop>
  <tabstop>seleccionaempresa</tabstop>
  <tabstop>configura</tabstop>
  <tabstop>nombreempresa</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
